lx_anonymizer/masking.py

```
@@ -0,0 +1,146 @@
+                    'ffmpeg', '-nostdin', '-y', '-i', str(input_video),
+                    '-vf', crop_filter,
+                    *encoder_args,
+                    '-c:a', 'copy',
+                    '-movflags', '+faststart',
+                    str(output_video)
+                ]
+                
+                logger.info(f"Direct crop masking (single pass) using {self.preferred_encoder['type']}: {crop_filter}")
+                logger.debug(f"FFmpeg command with -nostdin: {' '.join(cmd)}")
+                
+                result = subprocess.run(cmd, capture_output=True, text=True, check=True)
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)